addons/maptools/functions/fnc_canUseMapGPS.sqf

```diff
@@ -15,6 +15,10 @@
  * Public: No
  */
 
-visibleMap &&
-{alive ACE_player} &&
-{"ItemGPS" in (assignedItems ACE_player)}
+if (!visibleMap || {!alive ACE_player}) exitWith {false};
+
+private _gpsOpened = visibleGPS;
+private _gpsAvailable = openGPS true;
+if (!_gpsOpened) then {openGPS false};
+
+_gpsAvailable;
```

Comment:
```suggestion
_gpsAvailable // return
```
we omit final `;` on function returns

Reply:
Noted, but is the comment necessary? Don't remember seeing that anywhere else.

Reply:
It's been used in so many places in ACE3: https://github.com/search?q=repo%3Aacemod%2FACE3+%22%2F%2F+return%22+language%3ASQF&type=code